x-pack/plugins/observability_solution/dataset_quality/public/hooks/use_degraded_docs_chart.tsx

```diff
@@ -193,7 +192,7 @@ export const useDegradedDocsChart = ({ dataStream }: DegradedDocsChartDeps) => {
 };
 
 function getDataViewIndexPattern(dataStream: string | undefined) {
-  return dataStream ? `${indexNameToDataStreamParts(dataStream).type}-*-*` : DEFAULT_LOGS_DATA_VIEW;
+  return dataStream ?? DEFAULT_LOGS_DATA_VIEW;
```

Comment:
What I was referring to is to keep your change but instead of going for `DEFAULT_LOGS_DATA_VIEW` just use `${indexNameToDataStreamParts(dataStream).type}-*-*`

```suggestion
  return dataStream ?? `${indexNameToDataStreamParts(dataStream).type}-*-*`;
```

Reply:
whenever `dataStream` is not defined

Reply:
i just saw this comment, somehow this notification went on page 2 of notifications and i didn't visit it until today.
Good thing is i made dataStream mandatory. 

`Great minds think alike`